### PR TITLE
Add chefFlagOptions as env var

### PR DESCRIPTION
**Pull request by fadlinurhasan**
Add chefFlagOptions as env var

### CODE REVIEW

**Comment by giosakti** [comment]
@fadlinurhasan why not just name it bootstrapFlagOptions and reduce reference to specific bootstrapper (in this case chef) by as much as possible?

**Comment by fadlinurhasan** [comment]
> @fadlinurhasan why not just name it bootstrapFlagOptions and reduce reference to specific bootstrapper (in this case chef) by as much as possible?

Done Mas. Please check.

File: config/constant.go [hunk 1]
```diff
@@ -19,8 +19,9 @@ const (
 	DefaultPfMarkBootstrappedPath                 = "api/v2/node/containers/mark_bootstrapped"
 	DefaultPfMarkBootstrapErrorPath               = "api/v2/node/containers/mark_bootstrap_error"
 	DefaultPfMarkDeletedPath                      = "api/v1/node/containers/mark_deleted"
-	DefaultChefInstaller                          = "https://www.chef.io/chef/install.sh"
-	DefaultChefVersion                            = "15.5.17"
+	DefaultBootstrapInstaller                     = "https://www.chef.io/chef/install.sh"
```
Comment (giosakti) [inline config/constant.go]: can we also remove these 2 defaults about chef & chef version?
and put additional info on readme
@fadlinurhasan

Comment (fadlinurhasan) [reply config/constant.go]: Please take a look the changes mas https://github.com/pathfinder-cm/pathfinder-agent/pull/23/commits/9febc1817cc09f8e9d5611e8ee8932cc52da9d81

**Pull request merged**